lib/rules/sort-keys.js

```diff
@@ -105,13 +119,18 @@ module.exports = {
                         type: "integer",
                         minimum: 2,
                         default: 2
+                    },
+                    allowLineSeparatedGroups: {
+                        type: "boolean",
+                        default: false
                     }
                 },
                 additionalProperties: false
             }
         ],
 
         messages: {
+            allowLineSeparatedGroups: "'{{thisName}}' should be before '{{prevName}}'. To separate groups by line break, apply the allowLineSeparatedGroups option.",
```

Comment:
I'm not sure it's good to give option information to those who don't use this option.
I want to hear other member's opinion

Reply:
I think we should remove this message, and just use the existing one. I believe we don't have any error messages that advise users to reconfigure the rule.